File: ObjcCrashCourse/SuperCoolExercise/SuperCoolExercise/ViewController.m
//
//  ViewController.m
//  SuperCoolExercise
//
//  Created by Allen Czerwinski on 4/26/16.
//  Copyright © 2016 Allen Czerwinski. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIImageView *coolBackground;
@property (weak, nonatomic) IBOutlet UIImageView *coolLogo;
@property (weak, nonatomic) IBOutlet UIButton *buttonUI;

@end

@implementation ViewController



- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    _coolLogo.hidden = TRUE;
    _coolBackground.hidden = TRUE;
    _buttonUI.hidden = FALSE;
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)makeMeNotSoUncool:(id)sender {
    _coolBackground.hidden = FALSE;
    _coolLogo.hidden = FALSE;
    _buttonUI.hidden = TRUE;
}




@end

File: Swift2iOS9Udemy/HideTheBombsButtons/HideTheBombsButtons/ViewController.swift
//
//  ViewController.swift
//  HideTheBombsButtons
//
//  Created by Allen Czerwinski on 2/3/16.
//  Copyright © 2016 Allen Czerwinski. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var blueBombOutlet: UIImageView!
    @IBOutlet weak var redBombOutlet: UIImageView!
    
    @IBOutlet weak var reArmOutlet: UIButton!
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        reArmOutlet.hidden = true
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func hideBlueButton(sender: AnyObject) {
        blueBombOutlet.hidden = true
        reArmOutlet.hidden = false
    }
    @IBAction func hideRedButton(sender: AnyObject) {
        redBombOutlet.hidden = true
        reArmOutlet.hidden = false
    }
    
    @IBAction func reArmButton(sender: AnyObject) {
        blueBombOutlet.hidden = false
        redBombOutlet.hidden = false
        reArmOutlet.hidden = true
    }
    

}

